recipes/libsystemd/all/conanfile.py

```diff
@@ -0,0 +1,157 @@
+import functools
+import os
+import re
+
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibsystemdConan(ConanFile):
+    name = "libsystemd"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/systemd/"
+    description = "systemd library"
+    topics = ("conan", "systemd", "libsystemd")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_selinux": [True, False],
+        "with_lz4": [True, False],
+        "with_xz": [True, False],
+        "with_zstd": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_selinux": True,
+        "with_lz4": True,
+        "with_xz": True,
+        "with_zstd": True,
+    }
+    generators = "pkg_config"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        self.build_requires("meson/0.55.3")
+        self.build_requires("m4/1.4.18")
+        self.build_requires("gperf/3.1")
+        self.build_requires("pkgconf/1.7.3")
+
+    def requirements(self):
+        self.requires("libcap/2.45")
+        self.requires("libmount/2.36")
+        if self.options.with_selinux:
+            self.requires("libselinux/3.1")
+        if self.options.with_lz4:
+            self.requires("lz4/1.9.2")
+        if self.options.with_xz:
+            self.requires("xz_utils/5.2.5")
+        if self.options.with_zstd:
+            self.requires("zstd/1.4.5")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        src_folder = "systemd-stable-{}".format(self.version)
+        os.rename(src_folder, self._source_subfolder)
+
+    @property
+    @functools.lru_cache()
```

Comment:
`functools.lru_cache` is not available in python 2.7.
Yes, recipes of cci are supposed to build in 2.7.
```suggestion
```

Reply:
> Yes, recipes of cci are supposed to build in 2.7.

:fearful: How hard is to be alive in 2020.